web/services/health_summary_service.go

```diff
@@ -34,38 +34,46 @@ func (s *healthSummaryService) GetHealthSummary() (models.HealthSummary, error)
 	}
 
 	for _, sapSystem := range sapSystems {
-		var clusterIDs []string
 		var hostIDs []string
+		var clusterIDs []string
 
 		for _, instance := range sapSystem.GetAllInstances() {
-			clusterIDs = append(clusterIDs, instance.ClusterID)
 			hostIDs = append(hostIDs, instance.HostID)
+			clusterIDs = append(clusterIDs, instance.ClusterID)
 		}
 
-		clusters, err := s.clustersService.GetAll(&ClustersFilter{ID: clusterIDs}, nil)
+		hosts, err := s.hostsService.GetAll(&HostsFilter{ID: hostIDs}, nil)
 		if err != nil {
 			return nil, err
 		}
 
-		hosts, err := s.hostsService.GetAll(&HostsFilter{ID: hostIDs}, nil)
+		clusters, err := s.clustersService.GetAll(&ClustersFilter{
+			ID:          clusterIDs,
+			ClusterType: []string{models.SAPSystemTypeDatabase},
```

Comment:
I think this won't work. Here, we need to filter by `models.ClusterTypeHANAScaleUp`